BadHabbit/BadHabbit/Kit/DesignKit/Models/LayoutConstants.swift

```diff
@@ -15,4 +15,8 @@ enum LayoutConstants {
         static let height = 50
         static let bottomInset = 20
     }
+    enum ImageLogo {
+        static let heigth: CGFloat = 196
+        static let width: CGFloat = 196
```

Comment:
We can leave only 1 static let: named size, for example

Reply:
resolved